lib/Config.hpp

```diff
@@ -40,6 +40,7 @@ namespace Albinos
 
     std::optional<KeyWrapper> key;
     std::optional<KeyWrapper> roKey;
+    std::unique_ptr<std::vector<std::string>> settingNames;
```

Comment:
would be better to name it `settingsNames` to be consistent but it's not a big deal